src/schema/comments.ts

```diff
@@ -195,6 +215,15 @@ export const typeDefs = /* GraphQL */ `
       """
       first: Int
     ): CommentConnection!
+
+    """
+    Recommend users to mention in the comments
+    """
+    recommendedMentions(
+      postId: String!
+      name: String
```

Comment:
This naming makes more sense, given it's a search query in a way

```suggestion
      query: String
```